rotkehlchen/tests/unit/test_evm_contracts.py

```diff
@@ -36,11 +37,18 @@ def test_evm_abi_data(globaldb):
             abis_set.add(entry[1])
 
 
+def keep_open_globaldb_write():
+    """Silly way to keep a transaction open in another greenlet"""
+    with GlobalDBHandler().conn.write_ctx():
+        gevent.sleep(1.5)
+        gevent.sleep(1.5)
```

Comment:
why do you need two of them? To check that it changes and when coming back it keeps working?

Reply:
yes otherwise it can end before the contract.contracts() thing ends. Or at least that was my thinking. Is not really accurate science.